HOMEASSISTANT.md

```diff
@@ -1,6 +1,42 @@
 # Integration in Home Assistant
 Full home assistant integration isn't ready yet, but freakshock88 has found a way to do this:
-#### In automation.yaml::
+
+![HaCard](https://github.com/LaTrappe/nRF905-API/blob/master/images/HA_Card_Example.png)<br>
+
+#### In secrets.yaml:
+```
+ventilation_status: 'http://192.168.1.249/api/test/fan/querydevice.json'
+
+ventilation_turn_on_low: 'http://192.168.x.y/api/v2/fan/setspeed.json?speed=low'
+ventilation_turn_on_medium: 'http://192.168.x.y/api/v2/fan/setspeed.json?speed=medium'
+ventilation_turn_on_high: 'http://192.168.x.y/api/v2/fan/setspeed.json?speed=high'
+ventilation_turn_on_max: 'http://192.168.x.y/api/v2/fan/setspeed.json?speed=max'
+
+ventilation_turn_on_low_30_mins: 'http://192.168.x.y/api/v2/fan/setspeed.json?speed=low&timer=30'
+ventilation_turn_on_medium_30_mins: 'http://192.168.x.y/api/v2/fan/setspeed.json?speed=medium&timer=30'
```

Comment:
Under `rest_command` this is still called `ventilation_turn_on_medium_for_30mins`